aptible/resource_diff.go

```diff
@@ -0,0 +1,30 @@
+package aptible
+
+import "github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+type ResourceAttrs struct {
+	Required   []string
+	NotAllowed []string
+}
+
+type ResourceDiff struct {
+	*schema.ResourceDiff
+}
```

Comment:
Hmm, the primary reason I added this "alias" was to extend it with some additional methods because I didn't want the methods to be scoped to the `aptible` package but I suppose I could just make another package like `validation` and make the `ResourceDiff` a parameter on the `HasRequired` and `HasOptional` functions instead. What are your thoughts on that?